app/views/support_interface/provider_users/index.html.erb

```diff
@@ -13,6 +13,18 @@
       </li>
     </ol>
   </div>
+
+  <nav class="moj-sub-navigation" aria-label="Sub navigation">
+    <ul class="moj-sub-navigation__list">
+      <li class="moj-sub-navigation__item">
+        <%= govuk_link_to 'Support users', support_interface_support_users_path, class: 'moj-sub-navigation__link' %>
+      </li>
+
+      <li class="moj-sub-navigation__item">
+        <%= govuk_link_to 'Provider users', support_interface_provider_users_path, class: 'moj-sub-navigation__link', 'aria-current': 'page' %>
+      </li>
+    </ul>
+  </nav>
```

Comment:
Is it worth creating a Component for this which means the MOJ-specific CSS is only referred to in one place in the codebase?

Reply:
👍 done!